yarn.lock

```diff
@@ -6716,6 +6716,11 @@ no-case@^2.2.0, no-case@^2.3.2:
   dependencies:
     lower-case "^1.1.1"
 
+node-bin-setup@^1.0.0:
```

Comment:
do we need these new NPMs? i'd be very surprised! (similar comment for `package.json`)